RELEASE/scripts/autoscend/paths/standard.ash

```diff
@@ -1,8 +1,5 @@
 script "standard.ash"
 
-import<auto_util.ash>
```

Comment:
redundant import of auto_util.ash.
as both standard.ash and auto_util.ash are imported by autoscend.ash